notebooks/tutorial.raster_zonal_stats.ipynb

```diff
@@ -114,7 +114,7 @@
   },
```

Comment:
<p>Maybe move the recommendations to the very end for emphasis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>